pipe-cli/src/config.py

```diff
@@ -12,33 +12,133 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import click
+import base64
+from functools import update_wrapper
 import json
 import os
 import pytz
 import tzlocal
+from datetime import datetime, timedelta
 from pypac import api as PacAPI
 from pypac.resolver import ProxyResolver as PacProxyResolver
 
 from .utilities import time_zone_param_type
 
 PROXY_TYPE_PAC = "pac"
 PROXY_PAC_DEFAULT_URL = "https://google.com"
+ALL_ERRORS = Exception
+
 
 class ConfigNotFoundError(Exception):
     def __init__(self):
         super(ConfigNotFoundError, self).__init__('Unable to locate configuration or it is incomplete. '
                                                   'You can configure pipe by running "pipe configure"')
 
 
+def check_token(token, timezone, print_info=False):
```

Comment:
I would prefer not to make this file more complicated and move some methods into utilities as far as it possible

Reply:
@ekazachkova I agree, I moved the validation logic to the `access_token_validation.py` utility (e74e38ae60622e3f862f2084f8e6a49145263748)